templates/helpers.js

```diff
@@ -175,17 +175,16 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
                 return current;
             }
 
-            var splittedName = current.name.split('/');
-            var hasSlash = splittedName.length > 1;
+            var hasSlash = splitName.length > 1;
```

Comment:
@chasenlehara I think this will cause problems. You defined splitName on line 184, but we are using it here.